NUMPY/LINALG/EIGH/EIGH.py

```diff
@@ -9,6 +10,7 @@
 def EIGH(
     default: OrderedPair | Matrix,
     UPLO: str = "L",
+    select_return: Literal["w", "v"] = "w",
 ) -> OrderedPair | Matrix | Scalar:
```

Comment:
If I'm not mistaken, numpy nodes doesn't return OrderedPair, so it should be removed from the output type.